common/archive.h

```diff
@@ -164,58 +176,56 @@ class SearchSet : public Archive {
 	void add(const String& name, Archive *arch, int priority = 0, bool autoFree = true);
 
 	/**
-	 * Create and add a FSDirectory by name
+	 * Create and add a FSDirectory by name.
 	 */
 	void addDirectory(const String &name, const String &directory, int priority = 0, int depth = 1, bool flat = false);
 
 	/**
-	 * Create and add a FSDirectory by FSNode
+	 * Create and add a FSDirectory by FSNode.
 	 */
 	void addDirectory(const String &name, const FSNode &directory, int priority = 0, int depth = 1, bool flat = false);
 
 	/**
-	 * Create and add a sub directory by name (caseless).
+	 * Create and add a subdirectory by name (caseless).
 	 *
-	 * It is also possible to add sub directories of sub directories (of any depth) with this function.
-	 * The path seperator for this case is SLASH for *all* systems.
+	 * It is also possible to add subdirectories of subdirectories (of any depth) with this function.
+	 * The path seperator for this case is SLASH for all systems.
 	 *
-	 * An example would be:
+	 * Example:
 	 *
 	 *   "game/itedata"
 	 *
-	 * In this example the code would first try to search for all directories matching
+	 * In this example, the code first tries to search for all directories matching
 	 * "game" (case insensitive) in the path "directory" first and search through all
 	 * of the matches for "itedata" (case insensitive too).
 	 *
-	 * Note that it will add *all* matches found!
+	 * Note that it will add all matches found!
```

Comment:
This supposed to stress the word "all", by putting it in *italic*.

Reply:
I can revert this change if you want but honestly, whether we stress it or not, this sentence pretty much doesn't say anything else and it's impossible to miss the 'all'.